### PR TITLE
Imagestream limits

### PR DESCRIPTION
**Pull request by deads2k**
Prevents bypassing imagestream count limits.  Found a couple different bugs causing this.

@mfojtik or @miminar ptal

Fixes https://bugzilla.redhat.com/show_bug.cgi?id=1368078

### CODE REVIEW

**Comment by deads2k** [comment]
[test]

**Comment by mfojtik** [comment]
@deads2k rocks! thanks! 

LGTM

File: pkg/quota/image/imagestreamtag_evaluator.go [hunk 1]
```diff
@@ -48,13 +48,16 @@ func NewImageStreamTagEvaluator(istNamespacer osclient.ImageStreamTagsNamespacer
...
-		InternalOperationResources: map[admission.Operation][]kapi.ResourceName{admission.Update: computeResources},
-		MatchedResourceNames:       computeResources,
-		MatchesScopeFunc:           matchesScopeFunc,
-		UsageFunc:                  makeImageStreamTagAdmissionUsageFunc(isNamespacer),
-		GetFuncByNamespace:         getFuncByNamespace,
+		Name:              imageStreamTagEvaluatorName,
+		InternalGroupKind: imageapi.Kind("ImageStreamTag"),
+		InternalOperationResources: map[admission.Operation][]kapi.ResourceName{
+			admission.Update: computeResources,
+			admission.Create: computeResources,
```
Comment (miminar) [inline pkg/quota/image/imagestreamtag_evaluator.go]: So that's why! I forgot you added istag.Create() recently.

**Comment by miminar** [comment]
LGTM

**Comment by deads2k** [comment]
[merge]

**Pull request merged**